vllm/lora/layers.py

```diff
@@ -657,6 +685,53 @@ def apply_weights(self, x: torch.Tensor,
         return output
 
 
+class RefQKVParallelLinearWithLora(ColumnParallelLinearWithLoRA):
```

Comment:
Let's name this `QKVParallelLinearWithLora` and rename current `QKVParallelLinearWithLora` to `MergedQKVParallelLinearWithLora` (to be consistent with `ColumnParallelLinear`)

let's also add docstrings to both new layers